raisinbread/README.md

```diff
@@ -44,31 +44,30 @@ in the database. Ensure that a backup is available if this data is important:
 
 ```bash
 cat raisinbread/instruments/instrument_sql/9999-99-99-drop_instrument_tables.sql \
-    SQL/9999-99-99-drop_tables.sql | mysql
+    SQL/9999-99-99-drop_tables.sql | mysql -u admin_username -p database_name
```

Comment:
mmmm i think those details were previously omitted/underspecified because the _admin_username_ and possibly localhost argument mean the user can't just copy-paste this line. If you think this genuinely makes it clear for users and you've ensured it's consistent with how our other documentation ( docs/ ) invokes the mysql commandline, go for it.

Reply:
It was indeed omitted because the assumptions section above explains the command structure https://github.com/aces/Loris/pull/6937/files#diff-891d4b84b8f410eef353c238c8045ab6L35

Reply:
@ridz1208 Thank you for the comment! I have removed the changes.